Uppsala/UppEt10.xml

```diff
@@ -0,0 +1,445 @@
+                                            <ptr target="bm:Loefgren1974Katalog"/>
+                                            <citedRange unit="page">39–40</citedRange>
+                                            <citedRange unit="number">10</citedRange>
+                                        </bibl>
+                                        <bibl> 
+                                            <ptr target="bm:Zettersteen1899Upsala"/>
+                                            <citedRange unit="page">515–518</citedRange>
+                                            <citedRange unit="number">8</citedRange>
+                                        </bibl>
+                                        <bibl> 
+                                            <ptr target="bm:alvin"/>
+                                        </bibl>
+                                        <bibl> 
+                                            <ptr target="bm:BmWebsite"/>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    
+                </msDesc>
+                
+                <bibl> 
+                    <ptr target="bm:Loefgren1974Katalog"/>
+                    <citedRange unit="page">43–45</citedRange>
+                    <citedRange unit="number">14</citedRange>
+                </bibl>
+                <bibl> 
+                    <ptr target="bm:Zettersteen1899Upsala"/>
+                    <citedRange unit="page">519</citedRange>
+                    <citedRange unit="number">10</citedRange>
+                </bibl>
+            </sourceDesc>
+
+        </fileDesc>
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage><language ident="en">English</language></langUsage>
+            <langUsage><language ident="gez">Gəʿəz</language></langUsage>
+            <langUsage><language ident="gez-tr">Gəʿəz transliterated</language></langUsage>
+            <langUsage><language ident="de">German</language></langUsage>
+            <langUsage><language ident="sv">Swedish</language></langUsage>
+            <langUsage><language ident="am">Amharic</language></langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-07-25">Created record</change>
```

Comment:
I think you need to specify what you did!

Reply:
I personally do not specify when creating a record. I only specify for updates. I think, this is enough.